### ISSUE TITLE
renderCustomContent doesn't work

### ISSUE THREAD

**Issue by valery-lavrik** [lucasferreira/react-native-flash-message#109]
renderCustomContent parameter doesn't work.

```
showMessage({
	message: "Simple message",
	type: "info",
	renderCustomContent: ()=> <Text>Test renderCustomContent</Text>
});
```

only message is displayed

**Comment by rewaant** [comment]
Found a solution? In docs it says can't be added to a Message object in which case I'm assuming it'll be called on another FlashMessage instance. @lucasferreira is this the case? and do we have any example snippet?

Thanks!

**Comment by dan-lee** [comment]
This needs to passed as a prop on the `FlashMessage` component:
```tsx
<FlashMessage
  position="top"
  renderCustomContent={(options) => <Text>Test renderCustomContent</Text>}
/>
```

One problem I see is that the message is still rendered, even if you pass your own render method.

**Comment by Joshandrews43** [comment]
This works totally fine. No issues here, it was being called wrong by OP

**Comment by rewaant** [comment]
Still doesn't work for me. My setup: I'm using a global FlashMessage instance declared in App.js and using them in multiple screens/components. I've instantiated a local FlashMessage instance in a stackNavigator screen/component where I have to conditionally trigger a custom rendered component with showMessage but it won't work. I also tried removing the main instance in App.js and it still doesn't work.
